Sources/PerformanceTest/main.swift

```diff
@@ -21,7 +21,7 @@ let hostname = HBEnvironment.shared.get("SERVER_HOSTNAME") ?? "127.0.0.1"
 let port = HBEnvironment.shared.get("SERVER_PORT", as: Int.self) ?? 8080
 
 // create app
-let elg = MultiThreadedEventLoopGroup(numberOfThreads: 2)
+let elg = MultiThreadedEventLoopGroup(numberOfThreads: 8)
```

Comment:
Can we use `System.coreCount` here instead?

Reply:
Or `NIOSingletons.posixEventLoopGroup`

Reply:
Or `NIOSingletons.posixEventLoopGroup`

Reply:
I wanted to use a set number here, which doesn't use all the machines cores. I tend to use this with wrk for performance testing. I need to assign some cores to wrk otherwise it'll clash with the server and results will be inconsistent. 

Although that number should be 4 and not 8. Performance seems to top out at 4 cores on macOS for some reason. Possibly related to the limited number of connections you can make on macOS.